lib/cisco_node_utils/cmd_ref/encapsulation.yaml

```diff
@@ -0,0 +1,22 @@
+# encapsulation profile
+# encapsulation profile feature is available only on n7k
+---
+_exclude: [N3k, N5k, N6k, N9k]
+
+all_encaps:
+  multiple: true
+  get_command: "show encapsulation profile"
```

Comment:
Again, you need to have a good reason for not using the show run output to check cli state. It may be easier to get what you want from the non-show-run commands but believe me we've been burned many times on this.